src/lib/phdi-transforms/phdi_transforms/basic.py

```diff
@@ -0,0 +1,26 @@
+def transform_name(raw: str) -> str:
+    """trim spaces, capitalize, etc
+
+    >>> transform_name(" JohN Doe ")
+    'JOHN DOE'
+    """
+
+    raw = [x for x in raw if not x.isnumeric()]
+    raw = "".join(raw)
+    raw = raw.upper()
+    raw = raw.strip()
+    return raw
+
+
+def transform_phone(raw: str) -> str:
+    """Make sure it's 10 digits, remove everything else
```

Comment:
Have we considered what to do in the face of numbers that include a country code? I presume that this might not even be an issue since all of our data is assumed to come from the US, but even so we should state that somewhere.

Reply:
That's actually a really good call out. A lot of automated web pages that intake data (just as an example) automatically concatenate either `1` or `+1` to the front of phone numbers. Numbers that begin with a leading `+` are easy to identify a country code on, but we should have logic in place to handle an 11 digit number that starts with a leading 1. (Perhaps, if the original number had a delimiter between the first one and the next number, and the first one was alone, etc.)

Reply:
My vision for the next iteration of this functions is a more sophisticated process that uses a little regex to look for all the common phone number formates e.g +1(xxx) xxx-xxx, xxx-xxx-xxxx, etc... and parse accordingly. I don't think we need to worry about non-US yet. Patients count as cases in the state/country of their official residence not where they are infected/test positive.

Reply:
so what do we want to do for right now? Should we see what number formats look like in the actual data?

Reply:
I would say we're safe to merge in the 10-digit enforcement as is for the moment because we're not shipping anything yet, and then we write a 1 point fast follow ticket to investigate phone numbers in the real data and modify the logic here accordingly. Remember, our motto is "make it work draft, then iterate and improve"!